### ISSUE TITLE
Training result

### ISSUE THREAD

**Issue by lzr9926** [gentaiscool/end2end-asr-pytorch#2]
Could you tell me the accuracy of the trained model? The result of my training is very bad. Thanks！

**Comment by gentaiscool** [comment]
What command did you run?

**Comment by lzr9926** [comment]
My command line is:
python train.py --train-manifest data/libri_train_manifest.csv --val-manifest libri_val_manifest.csv --test-manifest libri_test_clean_manifest.csv --labels-path data/labels/labels.json --emb_cnn --shuffle --batch-size 10 --epochs 200 --cuda
Should I use a language model ? Will the result be better?
Thanks！

**Comment by gentaiscool** [comment]
The language model helps improving the performance. I haven't release my result on this repo. I will do it in the future especially on the full train set. 

Btw, did you use the full train set?

**Comment by lzr9926** [comment]
Not yet. I only used train-clean-100 to train the model. I also used Aishell Corpus to train the model, the result is not so good too.

**Comment by gentaiscool** [comment]
How did you evaluate your result? Using greedy or beam search?

**Comment by lzr9926** [comment]
Greedy. The result of CER is about 42%
